packages/cdktf/lib/terraform-module.ts

```diff
@@ -31,13 +32,20 @@ export abstract class TerraformModule
     super(scope, id);
 
     if (options.source.startsWith("./") || options.source.startsWith("../")) {
-      this.source = path.join("..", options.source);
+      throw new Error(
```

Comment:
I don't think this is going to work out very well. Users will expect to be able to use a relative path and it seems like something we should be able to support.